include/lsst/jointcal/Gtransfo.h

```diff
@@ -392,6 +394,31 @@ class GtransfoLinScale : public GtransfoLin {
     int getNpar() const { return 2; }
 };
 
+/**
+ * A Gtransfo that holds a SkyWcs
```

Comment:
Maybe note in the docstring what the primary usecase for this object is: holding the as-read WCS. Also maybe something about why it's not derived from BaseTanWcs.